CHANGELOG.md

```diff
@@ -4,8 +4,34 @@ Releases
 v0.4.0 (unreleased)
 -------------------
 
+This release requires regeneration of ThriftRW code.
+
+-   **Breaking**: Procedure registration must now always be done directly
+    against the `Dispatcher`. Encoding-specific functions `json.Register`,
+    `raw.Register`, and `thrift.Register` have been deprecated in favor of
+    the `Dispatcher.Register` method. Existing code may be migrated by running
+    the following commands on your go files.
+
+    ```
+    gofmt -w -r 'raw.Register(d, h) -> d.Register(h)' $file.go
+    gofmt -w -r 'json.Register(d, h) -> d.Register(h)' $file.go
+    gofmt -w -r 'thrift.Register(d, h) -> d.Register(h)' $file.go
+    ```
+-   **Breaking**: `Dispatcher.Register` now accepts a collection of
```

Comment:
As discussed offline - let's remove this. The Changelog should only contain changes affecting our end-users.